src/Providers/ElasticApmServiceProvider.php

```diff
@@ -200,4 +207,84 @@ protected function listenForQueries()
             app('query-log')->push($query);
         });
     }
+
+    protected function getCmdTransName($command)
+    {
+        return $command->command ? ('artisan ' . $command->command) : 'artisan help';
```

Comment:
I'm not sure if this is worth the effort, but have you considered projects based on Laravel Zero (https://github.com/laravel-zero/laravel-zero)? Those will not use the `artisan` command. That may be too much to implement now, but what about moving the `artisan` string to a variable so it could be set to a different value more easily in the future?